docs/awsome.md

```diff
@@ -0,0 +1,154 @@
+# The Helm Guide to Writing Awesome Charts
+
+A Helm Chart provides a recipe for installing and running a containerized application inside of Kubernetes. This guide explains how to write an outstanding Chart.
+
+## Creating a New Chart
+
+You can create a new chart using the `helm create` command. This will put the chart in your workspace, which is the perfect place for trying it out. You may choose to use the `helm edit` command to editor your chart, or you may be more comfortable editing directly with your favorite editor.
+
+## A Brief Anatomy of a Chart
+
+Inside of your newly created chart, there are three pieces:
+
+- The `Chart.yaml` file contains descriptive data about a chart.
+- The `manifests/` directory contains one or more Kubernetes manifest files.
+- A `README.md` file, which is a Markdown-formatted file that explains how to configure your chart.
+
+This guide walks you through the process of creating those parts.
+
+## Naming Your Chart
+
+A good name succinctly expresses what your chart provides. If your chart installs an application in the "normal" way or is a basic configuration of a package, you should name it with the name of the project. For example, a singe MongoDB chart may simple be called `mongo`.
+
+If your chart displays some more advanced features, or creates a cluster of servers, you should give it a more specific name: `redis-cluster` creates and configures a cluster or Redis servers.
+
+Please take a look at the existing charts before naming your own.
+
+_A Note on example charts:_ As Helm was getting started, we created a few example charts, whose purpose was to illustrate how to write a chart. We are trying to only add new _example_ charts when they illustrate something new and helpful for chart developers.
+
+## The Chart.yaml File
+
+A simple Chart.yaml file should look like this:
+
+```
+name: NAME OF CHART
+home: URL TO IMAGE HOME
```

Comment:
Side Note, how modular/monolithic are Charts expected to be? the `home` and `dependencies` fields pointf to the idea that a Chart should be modular.

I ask because a monolithic chart may have multiple image or project homes.